### ISSUE TITLE
Cannot switch selected part

### ISSUE THREAD

**Issue by InfinityPL** [daid/LegacyCura#653]
Windows 7 N x64
When I load more than 1 part I cant select them.
1 is highlighted in yellow - I can move it, but i can't select other parts.
Older versions of Cura were ok.

**Comment by daid** [comment]
Which version vs which version?

**Comment by InfinityPL** [comment]
13.11.2 (now on 64bit) vs 13.06.4 (then on 32bit)

**Comment by daid** [comment]
Odd...

Object selection works by drawing the screen in black with every object in different colors on it, reading back the pixel value below the mouse cursor. And then rendering the normal 3D view over that so you do not see the "selection" view.
Got any idea if anything in your configuration works against this? I know setting colors to 16bit instead of full color can cause this to fail. Maybe there are other problems that also impact this.

**Comment by sm7xvd** [comment]
I have this exact same issue. Version 14.09.

**Comment by CuraUser1** [comment]
I've had this problem with 2 versions of Cura, 14.06.1 and 14.09. The fix was to get the correct video driver. My laptop is a Dell E5500 which came with XP. I put Windows 7 on and even though the video worked seemingly fine, the video drivers were not correct. Dell's download site was not helpful and said Windows 7 wasn't supported on my model. I went to the Intel site and used their software to determine what driver I needed. I installed it, restarted, and I can now move multiple objects around the print area in Cura.

How I found this out was that I was using Draftsight and the mouse would be super sluggish in the drawing window only. I found the driver solution specifically for the Draftsight problem but thought maybe it was Cura's problem too. I checked immediately after installing the correct driver and everything works great now.
